.github/workflows/test-load-env-variables.yml

```
@@ -0,0 +1,28 @@
+name: Test load-env-variables
+
+on:
+  pull_request:
+    paths:
+      - "config/env/**"
+      - ".github/workflows/test-load-env-variables.yml"
+  push:
+    branches:
+      - v1
+    paths:
+      - "config/env/**"
+      - ".github/workflows/test-load-env-variables.yml"
+  workflow_dispatch:
+
+jobs:
+  load-env-variables-manual-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Load environment variables
+        uses: keep-network/ci/actions/load-env-variables@move-actions-to-ci #TODO: chage to @v1
```

We should use relative paths as the actions are located in this repository.
```suggestion
        uses: ./actions/load-env-variables
```

---

Applied in 6fbf7d82b80f72ff38c9dd1c74de3329868318e8 (I also changed the invocation of `upstream-builds-query` in the workflow testing that action).